src/Redux/Store/Expeditions/Expeditions/helpers/convertExpeditionFromConfig.ts

```diff
@@ -9,17 +11,32 @@ export const convertExpeditionFromConfig = (
   state: RootState
 ): types.Expedition => {
   const expeditionId = shortid.generate()
-
+  const seed = {
+    seed: config.seedConfig || shortid.generate(),
+    supplyState: true,
+    nemesisState: true,
+  }
+  const allMages = selectors.Settings.Expansions.Mages.content.getContent(state)
+    .ENG
+  const mageArray = Object.keys(allMages).map(function (id) {
+    let mage = allMages[id]
+    return mage
```

Comment:
you can just return `allMages[id]` here instead of making and intermediate assignment :)